eng/generators.targets

```diff
@@ -27,6 +27,12 @@
                             and @(Reference->AnyHaveMetadataValue('Identity', 'System.Memory')))
                           or ('@(ProjectReference)' != ''
                             and @(ProjectReference->AnyHaveMetadataValue('Identity', '$(CoreLibProject)'))))" />
+    <EnabledGenerators Include="DllImportGenerator"
+                       Condition="'$(EnableDllImportGenerator)' == ''
+                        and '$(IsFrameworkSupportFacade)' != 'true'
+                        and '$(IsSourceProject)' == 'true'
+                        and '$(MSBuildProjectExtension)' == '.csproj'
+                        and '$(TargetFrameworkIdentifier)' == '.NETStandard'" />
```

Comment:
Do we need to add a .NET Framework case as well? I know we discussed this a little offline, but I don't remember what conclusion we came to.

Reply:
I don't think so. I think this works as is, but will monitor it.